unlock-app/src/components/interface/checkout/main/Confirm.tsx

```diff
@@ -122,17 +122,23 @@ export function PricingData({ pricingData, lock, payment }: PricingDataProps) {
                 ) : null}
               </div>
 
-              <div className="font-bold">
-                {item.amount <= 0
-                  ? 'FREE'
-                  : payment?.route
-                  ? `${formatNumber(
-                      payment.route
-                        .convertToQuoteToken(item.amount.toString())
-                        .toFixed()
-                    ).toLocaleString()} ${symbol}`
-                  : `${formatNumber(item.amount).toLocaleString()} ${symbol}`}
-              </div>
+              {item?.creditCardAmount > 0 ? (
+                <div className="font-bold">
+                  {`$${formatNumber(item.creditCardAmount).toLocaleString()}`}
+                </div>
+              ) : (
+                <div className="font-bold">
+                  {item.amount <= 0
+                    ? 'FREE'
+                    : payment?.route
+                    ? `${formatNumber(
+                        payment.route
+                          .convertToQuoteToken(item.amount.toString())
+                          .toFixed()
+                      ).toLocaleString()} ${symbol}`
+                    : `${formatNumber(item.amount).toLocaleString()} ${symbol}`}
+                </div>
+              )}
```

Comment:
@julien51  

Here we show the credit card price for the single key in $USD, if present 

<img width="400" alt="Screenshot 2023-05-08 at 16 36 50" src="https://user-images.githubusercontent.com/20865711/236854658-69be45cb-b1bd-4914-a713-0d100d91d7c9.png">

otherwise, shoe the price by `symbol` price

<img width="400" alt="Screenshot 2023-05-08 at 16 19 58" src="https://user-images.githubusercontent.com/20865711/236854898-644e6b73-f313-4816-8ba8-a75c5c375ed5.png">

Reply:
I don't understand why we show it differently?
We should always show the $ prices here since that is what the user is going to pay. We should not show MATIC on that screen at all.

Reply:
> I don't understand why we show it differently? We should always show the $ prices here since that is what the user is going to pay. We should not show MATIC on that screen at all.

As for now, we always show the `symbol` and in the `credit card section` the price breakdown with the conversion.
I can change it and always keep the `$USD` price when converted or have credit card price